packages/components/src/local/organisms/support-panel/index.tsx

```diff
@@ -71,71 +71,102 @@ export const SupportPanel: React.FC<PropTypes> = ({
+  }, [opForces])
+
+  const SlideComponent = useMemo(() => (
+    <Slide direction="right" in={isShowPanel}>
+      <div className={styles.panel}>
+        <Rnd
+          disableDragging
+          style={PANEL_STYLES}
+          default={DEFAULT_SIZE}
+          minWidth={MIN_PANEL_WIDTH}
+          maxWidth={MAX_PANEL_WIDTH}
+          minHeight={MIN_PANEL_HEIGHT}
+          maxHeight={MAX_PANEL_HEIGHT}
+        >
+          <div className={styles.content}>
+            <TabPanel className={styles['tab-panel']} value={TABS[0]} active={activeTab === TABS[0]}>
+              {activeTab === TABS[0] &&
+                <PlanningAssets forceColors={forceCols} platformStyles={platIcons} forces={forces}
+                  playerForce={selectedForce} isUmpire={true} render={onRender} opFor={false}
+                  onSelectionChange={(data): void => onSelectionChange(false, data)} onVisibleRowsChange={(data): void => onVisibleRowsChange(false, data)} />
+              }
+            </TabPanel>
+            <TabPanel className={styles['tab-panel']} value={TABS[1]} active={activeTab === TABS[1]} >
+              {activeTab === TABS[1] &&
+                <PlanningMessagesList
+                  messages={messages}
+                  gameDate={gameDate}
+                  playerForceId={selectedForce}
+                  playerRoleId={selectedRole}
+                  isUmpire={true}
+                  icons={forceIcons}
+                  colors={forceCols.map((item: ForceStyle) => item.color)}
+                  names={forceNames}
+                  turnPresentation={turnPresentation}
+                  hideForcesInChannel={!!hideForcesInChannel}
+                  onRead={onRead}
+                  onUnread={onUnread}
+                  onMarkAllAsRead={onReadAll}
+                  channel={channel}
+                  templates={templates}
+                />
+              }
+            </TabPanel>
+            <TabPanel className={styles['tab-panel']} value={TABS[2]} active={activeTab === TABS[2]} >
+              {activeTab === TABS[2] &&
+                <PlanningAssets forceColors={forceCols} platformStyles={platIcons} forces={forces}
+                  playerForce={selectedForce} isUmpire={true} render={onRender} opFor={true}
+                  onSelectionChange={(data): void => onSelectionChange(true, data)} onVisibleRowsChange={(data): void => onVisibleRowsChange(true, data)} />
+              }
+            </TabPanel>
+            <div className={styles['resize-indicator-container']} >
+              <div className={styles['resize-indicator-icon']} >
+                <MoreVert fontSize='large' color='primary' style={{ marginLeft: 0 }} />
               </div>
             </div>
-          </Rnd>
-          <TabPanelActions onChange={onTabChange} />
-        </div>
-      </Slide>
+          </div>
+        </Rnd>
+        <TabPanelActions onChange={onTabChange} />
+      </div>
+    </Slide>
+  ),
+    [
+      isShowPanel,
```

Comment:
thanks @lebaphi - can I reduce the items in this list, to reduce the data change that trigger a re-render?

Reply:
Hi @IanMayo , yes you can.